config/docker/buildroot/Dockerfile

```
@@ -0,0 +1,31 @@
+FROM debian:buster-slim
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Docker for Jenkins really needs procps otherwise the Jenkins side fails
+RUN apt-get update && apt-get install --no-install-recommends -y procps
+
+ENV FORCE_UNSAFE_CONFIGURE=1
+
+# SSL / HTTPS support
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    apt-transport-https \
+    ca-certificates
+
+# Dependencies to run buildroot
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    bc \
+    bzip2 \
+    cpio \
+    gcc \
+    g++ \
+    git \
+    make \
+    rsync \
+    patch \
+    python3 \
+    file \
```

Alphabetical order tweak:
```suggestion
    cpio \
    file \
    gcc \
    g++ \
    git \
    make \
    rsync \
    patch \
    python3 \
```

---

fixed

---

build-and-push.sh changes tested locally